app/views/data_tables/plant_lines_table.html.haml

```diff
@@ -0,0 +1,10 @@
+= render 'data_tables/tabs'
+
+.lead Plant Lines table
+
+= link_to 'TEMP LINK TO Plant populations', data_tables_path(model: :plant_populations)
+
+= datatable_tag do
+  - PlantLine.table_columns.each do |table_column|
```

Comment:
I know, but as I mentioned in the PR description, I deliberately don't do it (yet).